environment.yml

```diff
@@ -8,5 +8,5 @@ dependencies:
   - pip:
     - voila>=0.5.0a2,<0.6.0
     - voila-material>=0.4.3
-    - jupyterlite>=0.1.0b18,<0.2.0
-    - jupyterlite-xeus-python
+    - jupyterlite-core[lab]>=0.1.0a1,<0.2.0
+    #- jupyterlite-xeus-python
```

Comment:
We'll probably need a proper release of `jupyterlite-core` and `jupyterlite` (0.1.0b19) first for the UI tests to pass.

Otherwise `jupyterlite-xeus-python` also brings the former (released) `jupyterlite` package which leads to the following issue: https://github.com/voila-dashboards/voici/actions/runs/4392929480/jobs/7692999441

![image](https://user-images.githubusercontent.com/591645/224494198-01da6b04-3f03-45cc-bd4b-dacf295b0192.png)


Looks fine after commenting `jupyterlite-xeus-python` (for now): https://github.com/voila-dashboards/voici/actions/runs/4393025973/jobs/7693153984

![image](https://user-images.githubusercontent.com/591645/224494219-49764d8a-eba8-48c3-ab0f-009c5b7473e0.png)